### ISSUE TITLE
Android app V2 - Devices turned on time wrong

### ISSUE THREAD

**Issue by Inversion-NL** [athombv/homey#1883]
**What is your Homey version (Settings → System)?**
1.5.3

**What did you try to achieve?**
See when a device is turned on in the new Android app

*Please include detailed steps how we can reproduce the issue*
Click on a device, look at turned on time

**What was the result?**
Time is apparently the same for all devices

**What did you expect as result?**
Exact time

**Comment by caseda** [comment]
FYI (though maybe a bit useless 😛) time is based on the time homey itself turned on.

**Comment by Inversion-NL** [comment]
Nope. PTP'd my Homey last night and the time is still exactly 13:37.
The video review of another user (available here: https://forum.athom.com/discussion/comment/60230/#Comment_60230)
Has the exact same time:
![image](https://user-images.githubusercontent.com/15921016/31491724-884ded38-af48-11e7-8b0b-985a8c1314c2.png)

So I guess your right: it could be a placeholder.

**Comment by Dijker** [comment]
is a not yet implemented  1337 as placeholer  ;-) https://en.wikipedia.org/wiki/Leet

**Comment by caseda** [comment]
Ah, I didn't even look at it that way 😛 calculator language ftw
